packages/client/base/src/types/hosted/index.ts

```diff
@@ -46,5 +48,39 @@ export type CheckoutProps = {
     display?: "same-tab" | "new-tab" | "popup";
     delivery?: "custodial" | "non-custodial" | "all";
     paymentMethods?: PaymentMethod[];
-}
+};
 export type SigninMethods = "metamask" | "solana";
+
+export const VerificationCollectionType = {
+    LOYALTY: "loyalty",
+    ART: "art",
+    MUSIC: "music",
+    GAMING: "gaming",
+    TICKETING: "ticketing",
+    CHARITY: "charity",
+    OTHER: "other",
+} as const;
+export type VerificationCollectionType = ObjectValues<typeof VerificationCollectionType>;
+
+type CrossmintVerificationFields = {
+    collectionType?: VerificationCollectionType;
+    collectionDescription?: string;
+    socials?: {
+        website?: string;
+        twitter?: string;
+        discord?: string;
+    };
+};
+
+export const VerificationProducts = {
+    PAYMENTS_CREDIT_CARD: "payments:credit-card",
+    PAYMENTS_CROSS_CHAIN: "payments:cross-chain",
+};
+export type VerificationProducts = ObjectValues<typeof VerificationProducts>;
```

Comment:
not sure if I'd call it products here, but not sure of a better name. VerificationTargets maybe?

Products I thought it was Items as in NFTs

Reply:
VerificationScopes probably better

Reply:
We got to discuss what approach to take designing this interface.

Approach 1: the SDK user to pass us the flows they want to perform (kyc, collection verification)
Approach 2: the SDK user sending us for what product / crossmint offering the need verification for (credit card payments, cross-chain payments)

I think we have much more control with Approach 2. Now the question is, how do we call "payments:credit-card", "payments:cross-chain". Product? Flow? Scope?

Just adding this comment for reference. I think scope makes sense.